pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -158,10 +158,11 @@ func (e *Signer) Run(ctx context.Context) {
 		return
```

Comment:
I suppose the function above will consumes the context, once the client-go refactoring goes through? 

I don't know what is easier from refactoring complexity POV to do/merge first. But this will result in some of the statement having the `bootstrap-signer` prefix and some statements that are printed via client-go without the prefix even though they belong to the same controller. 

Are we expecting to merge the client-go refactoring in the same release (1.27) ?

Reply:
the same goes for apimachinery and [ErrorHandlers](https://github.com/kubernetes/kubernetes/blob/1e3946ce9dcb745771b03845d4afd97caa7c7737/staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go#L87) for example. And other dependencies.

Reply:
Which client-go refactoring? Is that some specific effort or do you mean the migration of client-go to contextual logging?

I started that migration, but hadn't planned to submit it for 1.27 because some aspects of it a bit tricky and may require API changes. On the other hand you are right, unless all dependencies that log something also get migrated, a binary will have inconsistent logging. We are still in the alpha phase of contextual logging and this will not be noticeable unless someone explicitly flips the "ContextualLogging" feature gate, so that's okay?

Reply:
> Which client-go refactoring? Is that some specific effort or do you mean the migration of client-go to contextual logging?

I meant the contextual logging

> I started that migration, but hadn't planned to submit it for 1.27 because some aspects of it a bit tricky and may require API changes. On the other hand you are right, unless all dependencies that log something also get migrated, a binary will have inconsistent logging. We are still in the alpha phase of contextual logging and this will not be noticeable unless someone explicitly flips the "ContextualLogging" feature gate, so that's okay?

Thanks for explanation. I suppose that is fine if we are planning to wait with switching the gate on by default, until we can ensure a good experience.